namerd/iface/destination/src/main/resources/pull-destination-proto.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I'm no bash expert but https://www.shellcheck.net/ gives a few warnings about this script

Reply:
(Sorry for the drive-by review, but) I kind of wonder if this is something that we can do from sbt rather than with a bash script?

Reply:
@hawkw. I tried adding this LinkerdBuild.scala, but a bash script was easier. I can continue researching on how to do this through sbt and create a separate PR.

Reply:
it doesn't seem to me like figuring out how to add this to sbt is a good use of time